pinot-query-runtime/src/test/resources/queries/Comparisons.json

```diff
@@ -0,0 +1,481 @@
+{
+  "ints": {
```

Comment:
nit: this could be improvement points on the test framework side. 

but since we don't differentiate tables registered by different files. might be great to add a more unique name of the test such as `ints_comparison` this is to avoid registering 2 `ints_tbl` from 2 different test json files